docs/en/patterns/modal.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Modal
+table_of_contents: true
+---
+
+# Modal
+
+The modal pattern can be useful to overlay and area of the screen which can contain a prompt, dialog or interaction.
+
+<a href="https://vanilla-framework.github.io/vanilla-framework/examples/patterns/modal/"
```

Comment:
I believe this will be https://vanilla-framework.github.io/vanilla-framework/examples/patterns/modal/modal/.

You should only create an example directory if there is more then one example of a pattern.